float64 speed
string traj_type
---
bool success
